src/am_map.c

```diff
@@ -495,7 +495,7 @@ void AM_changeWindowLoc(void)
   incy = m_paninc.y;
   if (automaprotate)
   {
-    AM_rotate(&incx, &incy, -mapangle);
+    AM_rotate(&incx, &incy, mapangle);
```

Comment:
mapangle is `unsigned int`.

Reply:
Yes, but -90° is 270° and not the same as 90° - or am I missing something? Shoudn't this rather be `ANGLE_MAX - mapangle` then?

Reply:
> Shoudn't this rather be `ANGLE_MAX - mapangle` then?

Yes probably. MSVC told me that it's equivalent to `AM_rotate(&incx, &incy, mapangle);` and it somehow works anyway.

Reply:
So, does the compiler silently ignore the minus sign? I'd expect this to turn into `ANG270` somehow.

Reply:
> So, does the compiler silently ignore the minus sign?

Yes, there is the warning "C4146: unary minus operator applied to unsigned type, result still unsigned" From my testing, this works the same way with gcc.

I suggest this fix: `AM_rotate(&incx, &incy, viewangle - ANG90);`